ThunderCloud/NavigationController.swift

```diff
@@ -202,13 +205,13 @@ public extension UINavigationController {
 			guard let _url = url else { return }
 			
 			let safariViewController = SFSafariViewController(url: _url)
-			safariViewController.delegate = self
+			safariViewController.delegate = rightMostNavigationController
```

Comment:
Unsure about setting the delegate to `rightMostNavigationController` here!

Reply:
So I'm of the thinking that if we're presenting on a different navigation controller, we should set the delegate to be that same navigation controller - it feels wrong that we'd be presenting on one and then handing delegate methods elsewhere. What do you think? :)

Reply:
Ahhh I didn't realise it didn't do anything with the delegate methods :D That's fine then!